packages/flutter/lib/src/material/icon.dart

```diff
@@ -22,6 +24,14 @@ const Map<IconSize, int> _kIconSize = const <IconSize, int>{
   IconSize.s48: 48,
 };
 
+SplayTreeMap _kIconDensities = new SplayTreeMap.from({
+  1.0 : 'mdpi',
+  1.5 : 'hdpi',
+  2.0 : 'xhdpi',
+  3.0 : 'xxhdpi',
+  4.0 : 'xxxhdpi'
+});
```

Comment:
I was going to do this in a follow up PR pending a review of the overall design for AssetVendor.